### ISSUE TITLE
STAR OCEAN The Second Story R [010065301A2E0000]

### ISSUE THREAD

**Issue by hakushiii** [Ryujinx/Ryujinx-Games-List#4793]
### Game Name

STAR OCEAN: The Second Story R

### Game Update Version

1.0.0

### Emulator Build Version

1.1.1069

### Graphics Backend

Vulkan

### Hardware Specs

- ##### CPU: M2 8C
- ##### GPU: M2 8G
- ##### RAM: 8GB

### Game State

Remake of Star Ocean: The Second Story from 1998.
2D Characters with 3D Environment

Playable, stable at 30FPS with VSYNC On.
Crashes with VSYNC Off.

### Screenshots

![SO-TSS R1](https://github.com/Ryujinx/Ryujinx-Games-List/assets/58656012/b99c12a5-63a1-457f-9fa3-79831af24bdc)
![SO-TSS R2](https://github.com/Ryujinx/Ryujinx-Games-List/assets/58656012/f66a6074-ef6c-4df7-b272-d00eff4dcab7)
![SO-TSS R3](https://github.com/Ryujinx/Ryujinx-Games-List/assets/58656012/9b36de6c-4f7f-44ce-a6ec-0dfcf3acf2e0)
![SO-TSS R4](https://github.com/Ryujinx/Ryujinx-Games-List/assets/58656012/35c0eb02-01cc-4ea0-895f-0ab872b864e6)
![SO-TSS R5](https://github.com/Ryujinx/Ryujinx-Games-List/assets/58656012/d4b9de35-8f1d-476e-9c7c-ba8f059a2c76)
![SO-TSS R6](https://github.com/Ryujinx/Ryujinx-Games-List/assets/58656012/9e596d7e-bac5-44a3-b74f-43b2b43ea772)
![SO-TSS R7](https://github.com/Ryujinx/Ryujinx-Games-List/assets/58656012/c0a043fc-99db-4fef-b7f9-753d004edc1d)

### Log File

[Ryujinx_1.1.1069_2023-11-03_08-36-50.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/13245966/Ryujinx_1.1.1069_2023-11-03_08-36-50.log)

**Comment by PekaPete** [comment]
I would add that I have audio crackling issues when playing on my Mac (this happened in a few games that I've tried). This can be fixed with setting audio output to 96,000 Hz on the Mac's Audio MIDI Setup. Other than that, the game runs pretty smoothly with only slight stutters in the overworld map. Played for about 2 hours. My Mac is an M1 Pro 16" MBP with 16 GB RAM.

**Comment by hakushiii** [comment]
Yep, there's audio cracking. I was probably too busy jamming at the new music to notice :3

**Comment by Retsubty** [comment]
I have SEVERE audio cracking. Is there a fix for this? It's so severe that it sounds like the game is running very slow.

**Comment by hakushiii** [comment]
> I have SEVERE audio cracking. Is there a fix for this? It's so severe that it sounds like the game is running very slow.

head over to discord.gg/Ryujinx for support. See you there ^^

**Comment by dmjohn0x** [comment]
> I have SEVERE audio cracking. Is there a fix for this? It's so severe that it sounds like the game is running very slow.

This is fixed by going to settings > System then changing Audio Backend to SDL2.

**Comment by lostromb** [comment]
Pretty sure I got the same crash with an nvidia 980. I can try and narrow down the common factors on a few test systems (windows, linux, vulkan, AMD, etc.)

**Comment by starlitalpha** [comment]
Had a crash after a random battle on the overworld after leaving the first village while grinding. 
Made sure V-sync is on before starting.

Running windows 10, AMD Ryzen 7 5700X, EVGA RTX 3060 12GB version, 32 GB of ram. 
Nothing to do with "RDNA3 fuckery" as they so eloquently put it since I'm running an RTX 3060.
Seconding the "don't bother with the discord" statement. Only message I received was "You can't fix it."

[Ryujinx_1.1.1104_2024-01-12_17-23-50.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/13925305/Ryujinx_1.1.1104_2024-01-12_17-23-50.log)

**Comment by MetrosexualGarbodor** [comment]
The discord support channels are for issues you can actually troubleshoot. What other response do you expect if it's an issue that can't be troubleshooted? Should the staff waste your time instead?
The support channels are also not reporting or testing channels. That's what this repository is here for.

**Comment by starlitalpha** [comment]
> The discord support channels are for issues you can actually troubleshoot. What other response do you expect if it's an issue that can't be troubleshooted? Should the staff waste your time instead?

How about, "It's a known issue, and there is currently no fix." 
My time was wasted because there's no information here or the discord about this crash. There's only vague suppositions that it's because V-sync is off, it's "RDNA3 fuckery", and that maybe it's not. 
If I had known there was an unsolvable crash, I would have just stuck to the switch rather than try on Ryujinx. 
When people try to be as helpful as possible and they receive a dismissive and unhelpful response, it breeds contempt. 

To anyone else that comes to this page to check compatibility, allow me to make this clear. There are currently random crashes that cannot be fixed regardless of your GPU or if V-sync is off/on. Don't bother playing it on Ryujinx if that bothers you, or until you read/hear otherwise that the issue, whatever it is since we have no clue _what_ it is, has been fixed.

**Comment by dmjohn0x** [comment]
> > The discord support channels are for issues you can actually troubleshoot. What other response do you expect if it's an issue that can't be troubleshooted? Should the staff waste your time instead?
> 
> How about, "It's a known issue, and there is currently no fix." My time was wasted because there's no information here or the discord about this crash. There's only vague suppositions that it's because V-sync is off, it's "RDNA3 fuckery", and that maybe it's not. If I had known there was an unsolvable crash, I would have just stuck to the switch rather than try on Ryujinx. When people try to be as helpful as possible and they receive a dismissive and unhelpful response, it breeds contempt.
> 
> To anyone else that comes to this page to check compatibility, allow me to make this clear. There are currently random crashes that cannot be fixed regardless of your GPU or if V-sync is off/on. Don't bother playing it on Ryujinx if that bothers you, or until you read/hear otherwise that the issue, whatever it is since we have no clue _what_ it is, has been fixed.

Agreed, the discord staff are extremely unprofessional and outright jerks. Furthermore, mentioning this at all gets you marked as off-topic here, even though my initial post contained a direct answer to a question posed about audio issues posed in this very thread. I know this too will be hidden, as it likely should be, but I think when you have multiple people commenting on how rude and hateful the discord is, you should take note and try to do better.

Furthermore, if you guys know Ryujinx wont work with certain popular GPUs, that should be noted somewhere. If it's a game by game case, you can simply add a tag to games like this that states [Crashes with RDNA3] or something. Rather than getting hostile whenever people report this problem to you.

**Comment by lostromb** [comment]
I'm about 30 hours in-game now playing on generally recent versions of Ryujinx (I think mostly 1.1.1250 and later). Not a single crash, no audio issues, no apparent visual issues with the very rare exception of some vertex explosions which usually clear themselves up within a minute. Testing on Windows 10, Nvidia Titan X, Ryzen 2700X. You can also disable vsync to fairly easily get smooth 60fps with no drawbacks. A quick test on AMD-Windows also seems to run fine.

![STAR OCEAN THE SECOND STORY R_2024-05-22_20-11-22](https://github.com/Ryujinx/Ryujinx-Games-List/assets/918053/028fcef8-f234-4940-9503-19eb29803094)
![STAR OCEAN THE SECOND STORY R_2024-05-22_19-27-21](https://github.com/Ryujinx/Ryujinx-Games-List/assets/918053/1e38eab3-a564-4b22-9ef3-650dcdbbcf6c)
![STAR OCEAN THE SECOND STORY R_2024-05-22_20-13-46](https://github.com/Ryujinx/Ryujinx-Games-List/assets/918053/0f9af293-faee-4522-9422-0193546eab17)

**Comment by lostromb** [comment]
Request that status be changed to "Playable"
![Screenshot 2024-05-31 191420](https://github.com/Ryujinx/Ryujinx-Games-List/assets/918053/c633e4a6-1bd5-45f4-8a85-7ac75037a990)
